pom.xml

```diff
@@ -13,18 +13,18 @@
 		<project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
 		<!-- Dependencies -->
-		<version.xtf>0.29</version.xtf>
+		<version.xtf>0.35</version.xtf>
```

Comment:
hhmhm. What was the driving change for changing all thsoe versions?

Reply:
The largest change was the update to the fabric8io framework from a 5.x to a 6.x version. There were also some security issues I wanted to get addressed in the maven dependencies that should be resolved now too.